fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -848,6 +849,13 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
                 logger.error("Failed to fetch list of logins", e)
             }
         }
+
+        with(ShoppingSettings) {
```

Comment:
Sorry we might have had a discussion about this, can you explain what's happening here? I'm worried about having this in Application.

Reply:
I confirmed offline with @perrymcmanis144 that the `shopping.settings` metrics should be set at startup and attached to the metrics ping.

Reply:
Gotcha, thanks for the explanation. I now see other metrics being set here too